### PR TITLE
Add CSharpier package

### PR DESCRIPTION
**Pull request by mkonstapel**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.

My [package](https://github.com/mkonstapel/sublime-text-csharpier-plugin) is a wrapper for [CSharpier](https://csharpier.com).

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by mkonstapel** [comment]
> Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.

There is a `format_on_save` preference setting that's mentioned in the README, but it can just be added to Preferences.sublime-settings.

**Comment by mkonstapel** [comment]
`package-metadata.json` removed and LICENSE added :+1:

**Comment by braver** [comment]
Sorry for letting this sit for a while. It sounds useful and looks good.

Even if you have only one setting, please contain your settings in a settings file specific to your package and add the necessary UI for working with those settings. What you propose works of course, and would work for any setting in any package. But if all packages worked like that my preferences file would become very hard to manage. Users have other expectations, namely that packages have their own settings file. This also allows you to expose the setting and help users find it. Now they need to have seen it in the readme to even know that the setting exists.

File: repository/c.json [hunk 1]
```diff
@@ -4269,6 +4269,16 @@
 				}
 			]
 		},
+		{
+			"name": "CSharpier",
+			"details": "https://github.com/mkonstapel/sublime-text-csharpier-plugin",
```
Comment (braver) [inline repository/c.json:4274]: You might want to add labels like [formatting](https://packagecontrol.io/browse/labels/formatting)

Comment (mkonstapel) [reply repository/c.json:4274]: Done!

**Comment by mkonstapel** [comment]
> contain your settings in a settings file specific to your package and add the necessary UI for working with those settings

I don't know when I'll have time to figure out how to add that :-/ If that's a blocking issue, I'd prefer to remove the `format_on_save` option for now.

**Comment by braver** [comment]
This should help: https://github.com/mkonstapel/sublime-text-csharpier-plugin/pull/1

**Comment by mkonstapel** [comment]
> This should help: [mkonstapel/sublime-text-csharpier-plugin#1](https://github.com/mkonstapel/sublime-text-csharpier-plugin/pull/1)

You are too kind! Merged :+1:

**Comment by braver** [comment]
🫡

**Pull request merged**